codegenerator/cli/src/rescript_types.rs

```diff
@@ -423,13 +424,14 @@ impl RescriptTypeIdent {
         }
     }
 
-    pub fn to_rescript_schema(&self) -> String {
+    pub fn to_rescript_schema(&self, with_db_coerce: bool) -> String {
```

Comment:
It seems like horrible param tunnelling to just handle the one bigint case...

Can't we find a way to improve this? Maybe having explicit functions that don't take the boolean flag. Or just have a dedicated rescript type for the native version?

Also I don't like seeing boolean flags called in functions because you have to go to the definition to understand what they are for. Better to use an enum for the flag or wrap the bool in a new type like `struct WithDbCoerce(bool)`